cohorts/survival.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2016. Mount Sinai School of Medicine
```

Comment:
Pulled straight from checkpoint trials; seems like a reasonable home for now, until we have a separate plotting util?